tools/python/boutiques/schema/examples/example1/example1_docker.json

```diff
@@ -3,7 +3,8 @@
     "command-line": "exampleTool1.py [CONFIG_FILE] [STRING_INPUT_LIST] [STRING_INPUT] [FLAG_INPUT] [NUMBER_INPUT] [ENUM_INPUT] [FILE_INPUT] [LOG] [INT_LIST_INPUT]",
     "container-image": {
         "image": "boutiques/example1:test",
-        "type": "docker"
+        "type": "docker",
+        "container-opts": ["--hostname testHost"]
```

Comment:
One convention, is that spaces are omitted and items follow one another in a list, i.e. `['--hostname', 'testHost']`. I'm not sure how we feel about this/care to copy it.

Reply:
I think we can just join the elements in container-opts with a space.